test/modules/ModuleLib/NetworkController/NetworkControllerOperation.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil.NetworkController
+{
+    public enum NetworkControllerOperation
```

Comment:
should this be NetworkControllerOperationStatus? which will have "InProgress" and "Finished"?  Because it looks odd as we just do 1 operation and we report before and after, therefore the enum name should not be "Operation".

Reply:
I know this is something existing; I raise this as you touched this file, it is your call if you want to change it.

Reply:
I see your point. Let's change it in another PR, though